Shannon_Brown/python_stack/flask_fundamentals/checker/checker.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask, render_template  
+app = Flask(__name__)    
+@app.route("/")                        
+def index():
+	return "Welcome Home"
+         				 
+@app.route("/play")                                   
+def play():
+    return render_template ("playground.html")
+
+@app.route("/play/<var>")                                  
+def box_7(var):
+	v = int (var) 
+	return render_template ("playground2.html", num = v)
+
+@app.route("/play/<var>/<color>")                                 
+def box_5(var, color):
+	v = int (var)
+	return render_template ("playground3.html", num = v)
```

Comment:
you also need to pass color...eg render_template ("playground3.html", num = v, c = color)